package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "lru-queue",
-  "version": "0.1.0",
```

Comment:
Let's not add versions (I use [standard-version](https://github.com/conventional-changelog/standard-version#readme) for that

Reply:
fixed